packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -324,7 +330,7 @@ class Frame extends React.Component {
       if (this.state.isPositioned) {
         this.dropdown.removeAttribute('inert');
         this.dropdown.removeAttribute('aria-hidden');
-        document.querySelector(this.selectMenu).setAttribute('tabIndex', '0');
+        this.selectMenu.setAttribute('tabIndex', '0');
```

Comment:
I know this is not part of your code changes.. but menu has explicit [tabIndex=0](https://github.com/cerner/terra-core/blob/f937194e0b1d2e8b64bd4cabbbc0454d1d76cabb/packages/terra-form-select/src/combobox/Menu.jsx#L455)  so why do we need to set same tabIndex=0 again for menu.